backend/src/extension.ts

```diff
@@ -98,6 +98,25 @@ export class YeomanUIPanel {
 		return path.join(extensionPath, 'dist', 'media');
 	}
 
+	public async showOpenDialog(currentPath: string): Promise<string> {
+		let uri;
+		try {
+			uri = vscode.Uri.file(currentPath);
+		} catch (e) {
+			uri = vscode.Uri.file('/');
+		}
+
+		try {
+			const filePath = await vscode.window.showOpenDialog({
+				canSelectFiles: true,
+				defaultUri: uri
+			});
```

Comment:
In Theia 'prompt 2' of generator-foodq is empty.

![image](https://user-images.githubusercontent.com/57438361/75774953-b0ecc180-5d59-11ea-8d09-3e0b6453f825.png)